README.md

```diff
@@ -6,39 +6,111 @@ Please edit this README.md file with your answers to the worksheet questions.
+|    1    |    1    |    0    |    0       |    1         |    1     |    1                        |
+|    1    |    0    |    0    |    1       |    1         |    1     |    1                        |
+|    0    |    1    |    1    |    0       |    0         |    1     |    0                        |
+|    0    |    0    |    1    |    1       |    1         |    1     |    1                        |
+|    1    |    0    |    1    |    1       |    1         |    1     |    1                        |
+|    0    |    1    |    0    |    0       |    0         |    0     |    0                        |
 ### d
-
+|    A    |    B    |    C    |    D    |  NOT B   |   NOT C |  NOT A   |   NOT(B OR NOT C) |   NOT A AND D |   A AND NOT(B OR NOT C) AND (NOT A AND D) |
+|---------|---------|---------|---------|----------|---------|----------|-------------------|---------------|-------------------------------------------|
+|    0    |    0    |    0    |    0    |    1     |    1    |    1     |    1              |    0          |    0                                      |
+|    1    |    1    |    1    |    1    |    0     |    0    |    0     |    0              |    0          |    0                                      |
+|    1    |    1    |    1    |    0    |    0     |    0    |    0     |    0              |    0          |    0                                      |
+|    1    |    1    |    0    |    0    |    0     |    1    |    0     |    1              |    0          |    0                                      |
+|    1    |    0    |    0    |    0    |    1     |    1    |    0     |    1              |    0          |    0                                      |
+|    0    |    1    |    1    |    1    |    0     |    0    |    1     |    0              |    1          |    0                                      |
+|    0    |    0    |    1    |    1    |    1     |    0    |    1     |    1              |    1          |    0                                      |
+|    0    |    0    |    0    |    1    |    1     |    1    |    1     |    1              |    1          |    0                                      |
+|    0    |    1    |    1    |    0    |    0     |    0    |    1     |    0              |    0          |    0                                      |
+|    1    |    0    |    0    |    1    |    1     |    1    |    0     |    1              |    1          |    1                                      |
+|    0    |    1    |    0    |    0    |    0     |    1    |    1     |    1              |    0          |    0                                      |
+|    0    |    0    |    1    |    0    |    1     |    0    |    1     |    1              |    0          |    0                                      |
 ## Question 2
 
-### a
+![Logic Circuits](https://user-images.githubusercontent.com/56129572/67520335-8b132a00-f6a0-11e9-9601-f4858f6e55fa.png)
```

Comment:
These don't look like standard logic circuit diagrams -- it would be worth revising this topic. In general the "inner" parts of a boolean expression should be wired in as inputs to gates representing the "outer" parts